### ISSUE TITLE
Uploading screen broken when editing an issue

### ISSUE THREAD

**Issue by lleger** [GitHawkApp/GitHawk#2548]
Repro:
- Edit an existing issue
- Upload an image
- Screen doesn’t include save button to exit screen

Screenshot illustrating issue:
![GitHawk Upload by lleger](https://i.imgur.com/A2bKMJS.jpg)

**Comment by glynnis** [comment]
I'm also seeing this.

**Comment by rnystrom** [comment]
Is this on the TestFlight version? Also happening on the App Store?

If on TF, can you use FLEX to debug the view and see what's up?

FWIW I can't repro this

**Comment by lleger** [comment]
Happened on both versions. Happy to help debug but not sure what FLEX is?

**Comment by BasThomas** [comment]
In TestFlight builds, you can enable FLEX (an in-app debugger) with a three finger tap anywhere on the screen. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by lleger** [comment]
That is super cool. I’m not able to recreate this anymore with the latest TF version (maybe it was intermittent or got fixed in a recent build). I’ll use FLEX if it happens again and update here.

**Comment by lleger** [comment]
I was able to recreate this issue. When I had opened it originally, it was happening with a screenshot image, but that no longer seems to be an issue. However, I'm able to recreate this now consistently using a GIF. 

I'm happy to help debug some more but FLEX is no longer working for me with a three-finger tap.

**Comment by BasThomas** [comment]
Yeah; the current version is a golden master which has it disabled. 😬
